app/views/AssociatedCompaniesView.scala.html

```diff
@@ -46,7 +47,9 @@
                     errorLinkOverrides = Map(form.errors.head.key -> form.errors.head.key)))
         }
 
-        @AssociatedCompaniesViewHelper.heading(associatedCompaniesParameter)
+    <h1 class="govuk-heading-xl">
+        @messages(s"$heading")
```

Comment:
The value in `@heading` is a fully resolved message as a string so we don't want to wrap it again with messages